SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -545,38 +542,40 @@ void USpatialNetDriver::OnLevelAddedToWorld(ULevel* LoadedLevel, UWorld* OwningW
 	}
 }
 
-void USpatialNetDriver::OnAcceptingPlayersChanged(bool bAcceptingPlayers)
+void USpatialNetDriver::OnGSMUpdated()
 {
 	// If the deployment is now accepting players and we are waiting to spawn. Spawn.
-	if (bWaitingForAcceptingPlayersToSpawn && bAcceptingPlayers)
+	if (bWaitingForAcceptingPlayersToSpawn &&
```

Comment:
Should probably change this to `bWaitingToSpawn` since it's now based on `AcceptingPlayers` and `SessionID`

Reply:
Also, set `bWaitingToSpawn` to false when spawning via `OnMapLoaded`.